cypress/fixtures/docs/dynamicObjects.md

```diff
@@ -45,6 +46,26 @@ FIREBOLT_CALL = {
   content: ...
 };
 ```
+### How to initialize the Firebolt object
+- The Firebolt object should be initialized using the [the environment has been set up for {string} tests](../../support/step_definitions/testSetup.md#the-environment-has-been-set-up-for-string-tests) glue code.
```

Comment:
env has been setup glue code is a generic glue code that is used in all cases. It is not just applicable to getter/setter. You need to give details on each type of testType accepted - space, underscore, colon , dot. etc